packages/react/src/analytics/integrations/shared/GoogleConsentMode/GoogleConsentMode.ts

```diff
@@ -50,7 +26,7 @@ export class GoogleConsentMode {
    *
    * @returns Consent values for each category.
    */
-  private getConsentValues(consents: ConsentData) {
+  private getConsentValues(consents: ConsentData | null) {
```

Comment:
Treat consent as singular as it is like that everywhere else

```suggestion
  private getConsentValues(consents: ConsentData | null) {
```